libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/history/ReplayHistoryPlayer.kt

```diff
@@ -10,9 +11,12 @@ typealias ReplayEventsListener = (List<ReplayEventBase>) -> Unit
 /**
  * This class is similar to a music player. It will include controls like play, pause, seek.
  */
-class ReplayHistoryPlayer {
+class ReplayHistoryPlayer(
+    logger: Logger?
```

Comment:
Should we add a default value here (`null`) 🤔? We're adding it to `ReplayHistoryMapper`